src/raftstore/store/config.rs

```diff
@@ -79,6 +82,11 @@ pub struct Config {
     /// When a peer hasn't been active for max_peer_down_duration,
     /// we will consider this peer to be down and report it to pd.
     pub max_peer_down_duration: Duration,
+
+    /// When a peer has leader missing longer than max_leader_missing_duration,
+    /// It would ask pd to confirm whether it's still valid in some region.
+    /// If it's stale and out of any region, it will destroy itself.
```

Comment:
If the leader of a peer is absent for longer than max_leader_missing_duration, the peer will ask PD to confirm whether it is valid in any region. If the peer is stale and is not valid in any region, it will destroy itself.